pkg/kube/pod.go

```diff
@@ -235,9 +234,9 @@ func checkNodesStatus(p *v1.Pod, cli kubernetes.Interface) error {
 }
 
 // checkPVCAndPVStatus does the following:
-// 1. if PVC is present then check the status of PVC
-// 	1.1 if PVC is pending then check if the PV status is VolumeFailed return error if so. if not then wait for timeout.
-// 2. if PVC not present then wait for timeout
+//  1. if PVC is present then check the status of PVC
+//     1.1 if PVC is pending then check if the PV status is VolumeFailed return error if so. if not then wait for timeout.
+//  2. if PVC not present then wait for timeout
```

Comment:
This reformatting is showing that Go doesn't recognize the nested "1.1" item (Go doesn't support nested lists at all). The way this is formatted, it's actually treating it as a continuation of the `1.` list item — "… the status of PVC 1.1 if PVC is …".

We should probably reformat this to not use a nested list (i.e. move "1.1" to be a top-level item in the list, renumbering the items or switching to an unordered list).